test/baselines/adapter_JS_ES6Class_ESLint_TypeChecking_Spaces_SingleQuotes_Apache-2.0/.github/workflows/test-and-release.yml

```diff
@@ -3,156 +3,80 @@ name: Test and Release
 # Run this job on all pushes and pull requests
 # as well as tags with a semantic version
 on:
-    push:
-        branches:
-            - "*"
-        tags:
-            # normal versions
-            - "v[0-9]+.[0-9]+.[0-9]+"
-            # pre-releases
-            - "v[0-9]+.[0-9]+.[0-9]+-**"
+  push:
```

Comment:
You switched from 4 to 2 spaces for this yml file. Is this going to cause an issue with prettier or so? Is this consistent with other yml files?

Reply:
We currently exclude the `.yml` files from Prettier. I've found 4 spaces to be unnecessarily wide in yml files. The Dependabot auto-merge workflow and Dependabot config already use 2 spaces, the Dependabot auto-merge config uses 4.